checker-plugin/src/main/scala/org/virtuslab/akkasaferserializer/AkkaSerializabilityCheckerPlugin.scala

```diff
@@ -7,10 +7,12 @@ class AkkaSerializabilityCheckerPlugin(override val global: Global) extends Plug
   override val name: String = "akka-serializability-checker-plugin"
   override val description: String =
     """checks whether a specified Akka serialization is applied to all messages, events and persistent state classes"""
+
```

Comment:
I'd also rename the class to `AkkaSerializabilityCheckerCompilerPlugin` for clarity b/w sbt and compiler plugins... this might seem obvious now but won't be so in a few months-a year :///

Reply:
#48